cumulus/pallets/collator-selection/src/lib.rs

```diff
@@ -982,3 +985,16 @@ pub mod pallet {
 		}
 	}
 }
+
+/// TypedGet implementaion to get the AccountId of the StakingPot.
+/// QUESTION: is there a better way to do this?
+pub struct StakingPotAccountId<R>(PhantomData<R>);
+impl<R> TypedGet for StakingPotAccountId<R>
+where
+	R: crate::Config + frame_system::Config,
+{
+	type Type = <R as frame_system::Config>::AccountId;
+	fn get() -> Self::Type {
+		<crate::Pallet<R>>::account_id()
+	}
+}
```

Comment:
@muharem this was the only way I was able to use `ResolveTo` instead of `ToStakingPot`, because it takes the account as a `TypedGet`. 

Let me know if you know of a cleaner way to do this.